packages/firestore/lite/src/api/reference.ts

```diff
@@ -302,6 +303,28 @@ export function collection(
   }
 }
 
+export function collectionGroup(
+  firestore: firestore.FirebaseFirestore,
+  collectionId: string
+): Query<firestore.DocumentData> {
+  const firestoreClient = cast(firestore, Firestore);
+
+  validateArgType('collectionGroup', 'non-empty string', 1, collectionId);
+  if (collectionId.indexOf('/') >= 0) {
+    throw new FirestoreError(
```

Comment:
recommend using an ErrorFactory https://github.com/firebase/firebase-js-sdk/blob/master/packages/util/src/errors.ts#L106

Reply:
I was unaware of this Added TODO.